contracts/RapidCoin.sol

```diff
@@ -234,79 +262,71 @@ contract RapidCoin is StandardToken {
     uint256 preSaleEndDate;
     uint256 saleStartDate;
     uint256 saleEndDate;
-
+    address walletForBalanace;
     //TODO presale 4 weeks but full sale start after 4 week
     function RapidCoin () public {
         totalSupply = 10 * (10 ** 9) * (10 ** decimals); //10 billion
         tokenSaleLimit = 475 * (10 ** 7) * (10 ** decimals); //4.75 billions
-        uint256 ownerTokens = totalSupply ; //we assign 10 million to acccountant firstly. 
+        uint256 ownerTokens = totalSupply; //we assign 10 million to acccountant firstly. 
         balances[msg.sender] = ownerTokens;
-        amountRaised = 0;
         owner = msg.sender;
+        walletForBalanace = msg.sender;
```

Comment:
@yasiriqbal776 is this a part of business requirements? Transfer all ether to contract owner?

Reply:
No this address will be change by admin to receive ether , currently they didnot provide so i forward balance to owner.